Green500 Measurement Example RWTH

chriswasser opened the pull request:
As per issue #104, this PR provides a description of the two recent submissions of the RWTH Aachen University for the new CLAIX-2023 cluster. An overview table precedes the specific examples to ease discoverability and navigation through the different sections. A few LaTeX packages were added for improved table and unit formatting, creation of clickable links, and acronym management.

tilsche reviewed (commented):
Thank you for the excellent PR. I really like the overview table also as a starting point for hopefully more examples to come. There are only some minor comments:

- I'm not sure if we want a separate acronyms section given that we already have the definitions. Do you see a way to merge the two gracefully, perhaps if we accept that the existing entries are not yet consitently linked?
- Can you, with reasonable effort, share the data behind Figure 8? Either by reference to a reproducibility package or for inclusing in the source tree?

chriswasser commented:
Just added new commits to address the requested changes:

- The raw data including a data processing and visualization script is now included within the data directory.
- The specified powermeter accuracy is now split into the percent of the measured value and the percent of the measuring range.
- A separate acronym and definition section was kept because, based on quick online searches, it seems quite common to have both these sections separat in the same document. If one was still inclined to combine these sections, the glossaries package would be an option. However, the resulting PDF output can look a little odd in the following cases: A) When specifying a definition for a word, which does not have a short form, or B) when specifying an acronym for a word, which does not have a description (see both cases below). Further customizations to fix these visual oddities would probably require further manual LaTeX scripting - I would leave this to a future PR for now.

![glossaries-package-output](https://github.com/user-attachments/assets/fe3eb462-a7a3-4499-bc4e-948ba5ba9981)

Hope this resolves the addressed points and enables a merge into the main document 🤓

tilsche commented:
> * A separate acronym and definition section was kept because, based on quick online searches, it seems quite common to have both these sections separat in the same document. If one was still inclined to combine these sections, the glossaries package would be an option. However, the resulting PDF output can look a little odd in the following cases: A) When specifying a definition for a word, which does not have a short form, or B) when specifying an acronym for a word, which does not have a description (see both cases below). Further customizations to fix these visual oddities would probably require further manual LaTeX scripting - I would leave this to a future PR for now.

I agree. Let's get this merged :-)
